BraveShared/Analytics/DAU.swift

```diff
@@ -38,27 +38,32 @@ public class DAU {
             return false
         }
         
+        // Sending ping immediately
+        sendPingToServerInternal()
```

Comment:
To be insanely safe, I think we should maybe perform a ping on app open too... Feels kind of redundant with a 5 minute timer, but this is technically what Android does.

Reply:
https://github.com/brave/brave-ios/pull/444/files#diff-19fd1a6fe4b2f78eb457726f366733e0R285
See it here, now, thx